src/Controller/QueueTrigger.php

```diff
@@ -36,4 +42,33 @@ public function processToEndTime() {
       return $build;
     }
   }
+
```

Comment:
Is any of this generator needed?   Seems like unnecessary thrashing.

Reply:
Nah, it's fluff, @swirtSJW - I'm removing.